examples/nodejs/eventbridge/webapp/src/App.tsx

```diff
@@ -0,0 +1,172 @@
+import DescribeCache from "./components/DescribeCache";
+import Topic from "./components/Topic";
+import { clearCurrentClient, Message, subscribeToTopic } from "./utils/momento-web";
+import type { TopicItem, TopicSubscribe } from "@gomomento/sdk-web";
+import { ArrowDown } from "./svgs/arrow-down";
+import DeleteRecordForm from "./components/DeleteRecordForm";
+import { ArrowLeft } from "./svgs/arrow-left";
+import { ArrowRight } from "./svgs/arrow-right";
+import InfoModal from "./components/InfoModal";
+
+const App = () => {
+  const [location, setLocation] = useState("");
+  const [maxTemp, setMaxTemp] = useState("");
+  const [minTemp, setMinTemp] = useState("");
+  const [precipitation, setPrecipitation] = useState("");
+  const [isSubscribedToTopic, setIsSubscribedToTopic] = useState<boolean>(false);
+  const [messages, setMessages] = useState<Message[]>([]);
+  const [isInfoModalVisible, setIsInfoModalVisible] = useState<boolean>(() => {
+    const infoModalClosed = localStorage.getItem("isInfoModalClosed");
+    return infoModalClosed !== "true";
+  });
+  const handleSetMessages = (message: Message) => {
+    setMessages((curr) => [...curr, message]);
+  };
+
+  const onItem = (item: TopicItem) => {
+    try {
+      const message = {
+        text: item.valueString(),
+        timestamp: Date.now(),
+      };
+      handleSetMessages(message);
+    } catch (e) {
+      console.error("unable to parse message", e);
+    }
+  };
+
+  const onError = async (
+    error: TopicSubscribe.Error,
+    sub: TopicSubscribe.Subscription,
+  ) => {
+    console.error(
+      "received error from momento, getting new token and resubscribing",
+      error,
+    );
+    sub.unsubscribe();
+    clearCurrentClient();
+    await subscribeToTopic(onItem, onError);
+  };
+
+  useEffect(() => {
+    clearCurrentClient();
+    subscribeToTopic(onItem, onError).then((subscription) => {
+      if (subscription) {
+        setIsSubscribedToTopic(true);
+      }
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { id, value } = event.target;
+    if (id === "location-input") {
+      setLocation(value);
+    } else if (id === "max-temp-input") {
+      setMaxTemp(value);
+    } else if (id === "min-temp-input") {
+      setMinTemp(value);
+    } else if (id === "precipitation-input") {
+      setPrecipitation(value);
+    }
+  };
+
+  return (
+    <div className="flex flex-col h-screen bg-gray-50">
+      <header className="bg-green-900 p-4 text-white">
+        <div className="flex justify-center items-center">
+          <img src={MomentoLogo} alt="Momento Logo" className="w-8 h-8" />
+          <span className="mx-2">+</span>
+          <img src={AwsEventBridgeLogo} alt="AWS EventBridge Logo" className="w-8 h-8" />
+        </div>
+        <h1 className="text-lg font-semibold text-center mt-2">Welcome to Weather Statistics Demo</h1>
```

Comment:
Can we maybe change this text to `Welcome to the DynamoDB-Momento EventBridge Demo`?